content/src/migrations/scripts/1663100676618_snapshots.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { ColumnDefinitions, MigrationBuilder } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('snapshots', {
+    hash: { type: 'text' },
+    init_timestamp: { type: 'timestamp', primaryKey: true, notNull: true },
+    end_timestamp: { type: 'timestamp', primaryKey: true, notNull: true },
+    replaced_hashes: { type: 'text[]', notNull: true },
```

Comment:
is this correct? Couldn't it be that it didn't replace anything? Or the empty list is not null?